src/gui/sharemanager.cpp

```diff
@@ -540,6 +540,11 @@ void ShareManager::fetchShares(const QString &path)
     connect(job, &OcsShareJob::shareJobFinished, this, &ShareManager::slotSharesFetched);
     connect(job, &OcsJob::ocsError, this, &ShareManager::slotOcsError);
     job->getShares(path);
+
+    auto *sharedWithMeJob = new OcsShareJob(_account);
```

Comment:
```suggestion
   const auto sharedWithMeJob = new OcsShareJob(_account);
```

Reply:
none of the methods in `OcsShareJob` are declared `const` (due to `setVerb`/`addParam` and the likes), so calling the job won't compile ...

Reply:
true, good catch!  will also adapt the `auto *` above then